addons/lunch/report/lunch_product_report.py

```diff
@@ -66,6 +66,7 @@ def write(self, values):
     def init(self):
         tools.drop_view_if_exists(self._cr, self._table)
 
+        self._cr.execute("""CREATE INDEX lunch_order_user_product_date ON lunch_order (user_id, product_id, date)""")
```

Comment:
Should be defined in `lunch.order` model.

Reply:
And uses `CREATE INDEX ... IF NOT EXISTS ON ...`

Reply:
Nice catch :D